### DISCUSSION TITLE
Columns width in "import traces"

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1388] [General]
Hello
Is there any way to stabilize column width in "Import traces". Width changes all the time from a session to another. I adjust size automatically several times, but it doesnt do anything
Thank you

**Comment by wolfgang-ch** [comment]
Which view contains "Import trace", can you provide a screenshot.

You have not yet provided a screenshot for this issue: https://github.com/mytourbook/mytourbook/discussions/1380

**Comment by gringojack** [comment]
![Error](https://github.com/user-attachments/assets/6c521d4b-fa38-49ed-8b56-58661b95a2d5)
At times this error pane shows up

https://github.com/user-attachments/assets/9c440584-a0a8-47fa-beff-5a6da8a412e2

**Reply by wolfgang-ch** [reply]
## SQL Exception
Can you provide the log

## Columns

The column widths in this view and in many other views are saved automatically when the app or view is closed. You may test column profiles, open the column context menu by hovering the mouse over the column header

![image](https://github.com/user-attachments/assets/ecf8d9f4-2ba5-4b07-a582-0dae1011a732)

**Reply by gringojack** [reply]
OK but where can I find the SQL exception log?

**Reply by FJBDev** [reply]
To me, this error seems to be the same error when MT is [launched already](https://github.com/mytourbook/mytourbook/issues/1220) ?

> ![Error](https://private-user-images.githubusercontent.com/4082252/356161616-6c521d4b-fa38-49ed-8b56-58661b95a2d5.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMxMzkyNjIsIm5iZiI6MTcyMzEzODk2MiwicGF0aCI6Ii80MDgyMjUyLzM1NjE2MTYxNi02YzUyMWQ0Yi1mYTM4LTQ5ZWQtOGI1Ni01ODY2MWI5NWEyZDUuanBnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MDhUMTc0MjQyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjI3ODNhZDQ1MzE5MzA2NWM0NWZkMDE2NGY4N2MxNzA1ZGQyYTc2MTliOWJiMjc4ZDg2NGY1MDIyZTVlODk0NiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.6eFVp-gh0IHX2VYBEPibMcX5huCBsekyFohO6uCJWDU) At times this error pane shows up

**Comment by wolfgang-ch** [comment]
>  Is there any way to stabilize column width in "Import traces". Width changes all the time from a session to another. I adjust size automatically several times, but it doesnt do anything

With my new 4k display I can confirm that this issues happens also when you add or remove a column. I think this is a 4k display issue. I just wanted to do the the final tests for MT 24.8 and publish it today but firstly I will fix this issue.

In the above video I can see that you are using also a 4k display because the toolbar icons are tiny which can be fixed with this hint https://github.com/mytourbook/mytourbook/discussions/1389

**Comment by wolfgang-ch** [comment]
I created a new issue https://github.com/mytourbook/mytourbook/issues/1390
